models/articles.models.js

```diff
@@ -0,0 +1,24 @@
+const connection = require('../db/connection')
+
+const allArticlesById = (article_id) => {
+    return connection.query(`SELECT * FROM articles WHERE article_id = $1`, [article_id])
+    .then(({rows}) => {
+        return rows[0] 
+    })
+}
+
+const allArticles = () => {
+    return connection.query(`SELECT articles.author, articles.title,articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.*) AS comment_count 
+    FROM articles 
+    LEFT JOIN comments
+    ON articles.article_id = comments.article_id
+    GROUP BY articles.article_id
+    ORDER BY created_at desc`)
+    .then(({rows}) => {
+        return rows 
+    })
+}
```

Comment:
Well done on this, it's quite a complicated SQL statement!

It looks like you're counting the comments which is good, it's just you aren't checking it in your testing - see my comment in your test file for more info on this.